### PR TITLE
更新 options.py

### PR DESCRIPTION
**Pull request by 71n9**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

```python
options = webdriver.ChromeOptions()
options.timeouts = {"pageLoad": 60}
# ValueError: Timeout keys can only be one of the following: implicit, pageLoad, script
```
I want to set the pageLoad timeout
When I execute the above code, I receive an error

### Description

A small change

### Motivation and Context

You can set successfully after modifying the logic

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by diemol** [comment]
You set it in the driver after creating it: `driver.set_page_load_timeout(30)`, for example.

**Comment by 71n9** [comment]
> You set it in the driver after creating it: `driver.set_page_load_timeout(30)`, for example.

```python
# Part of the code of my program

options = webdriver.ChromeOptions()
options.add_experimental_option("debuggerAddress", f"127.0.0.1:{self.port}")
options.timeouts = {"pageLoad": 60000}
try:
   self.chromeDriver = webdriver.Chrome(service=Service(ChromedriverPath), options=options)
except Exception as err:
    print(err) # Message: timeout: Timed out receiving message from renderer: 300.000

```
image

I'm trying to solve this problem
I must set the timeout only before receiving this error, or I will have to wait 300 seconds to continue execution

**Comment by diemol** [comment]
That is different, you seem to be having issues creating the session. First try without this
`options.add_experimental_option("debuggerAddress", f"127.0.0.1:{self.port}")`, and then debug from that point.

If you have more questions, please use the following channels https://www.selenium.dev/support/

**Comment by 71n9** [comment]
> 
Running for a long time will cause chrome to get stuck


> 那是不一样的，你好像在创建会话时遇到问题。首先尝试不使用this `options.add_experimental_option("debuggerAddress", f"127.0.0.1:{self.port}")`，然后从那一点开始调试。
> 
> 如果您有更多问题，请使用以下渠道https://www.selenium.dev/support/

The session has been running for a long time Running for a long time will cause chrome to get stuck
All GUI functions of Chrome are invalid The manual shutdown procedure also fails
I have to force the end of chrome and restart the remote binding
Each re-link must wait 300 seconds

**Comment by titusfortner** [comment]
@diemol we should encourage setting timeout values in options instead of after session starts.

@symonk can you check this logic? I can't tell what we support now vs what this code allows.

**Comment by diemol** [comment]
I see. I was checking the docs and saw something else. Good to know.

**Comment by titusfortner** [comment]
It's one of the things added with w3c that probably needs to be updated in our docs in multiple places.

**Comment by 71n9** [comment]
The matter involved in the end should be very complex
Should lose a few hairs
thank you for all you do Thank you for open source software

File: py/selenium/webdriver/common/options.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def timeouts(self, timeouts: dict) -> None:
 
         :param timeouts: values in milliseconds for implicit wait, page load and script timeout
         """
-        if all(x in timeouts.keys() for x in ("implicit", "pageLoad", "script")):
+        if all(x in ("implicit", "pageLoad", "script") for x in timeouts.keys()):
```
Comment (symonk) [inline py/selenium/webdriver/common/options.py:130]: Thanks, I think this is a slightly better implementation than what existed before (Enforcing every timeout was provided else giving an ambiguous error) to now making sure all keys are in the set list, this is ok for now.  In future I think we should probably extract the 'known' keys and throwaway the rest without causing an error).

**Review by symonk — APPROVED** [review]

**Comment by symonk** [comment]
Thanks, I think this is a slightly better implementation than what existed before (Enforcing every timeout was provided else giving an ambiguous error) to now making sure all keys are in the allowed sequence, this is ok for now.  In future I think we should probably extract the 'known' keys and throwaway the rest without causing an error).

**Pull request merged**